spec/javascripts/test_spec.js

```diff
@@ -57,6 +57,27 @@ describe("Basic testing", function() {
     blurredLocation.setBlurred(false);
     expect(blurredLocation.isBlurred()).toBe(false);
   });
+
+  it("Checks if input listeners change maps position to the entered latitude and longitude", function() {
+
+    var lat = document.getElementById("lat");
+    var lng = document.getElementById("lng");
+
+    lat.value = 20;
```

Comment:
Here you'll need (in jQuery at least) `latEl.val(20)` -- and please rename these `latEl` etc so we know they're DOM elements. Thanks! Very close!